docs/api/supervisor/endpoints.md

```diff
@@ -36,9 +36,9 @@ Return overview information about installed add-ons.
       "version": null,
       "version_latest": "1.0.1",
       "update_available": false,
-      "installed": false,
```

Comment:
Is this a breaking change for api users? Do we talk about breaking changes for add-on api in developer blogs?

Reply:
Yes and no, this is something that should have been a part of https://github.com/home-assistant/supervisor/pull/3623